hasher-matcher-actioner/src/OpenMediaMatch/storage/interface.py

```diff
@@ -22,6 +22,7 @@
 
 import flask
 
+from threatexchange.cli.storage.interfaces import IContentTypeConfigStore
```

Comment:
I think you are getting a mypy failure here because this might need a missing `__init.py__` file in the interfaces directory

Reply:
i don't know why mypy keeps complaining about not finding the module. I have checked: 
* threatexchange package is installed 
* `threatexchange.storage.interfaces` is a valid module
* has `__init__.py` in interfaces directory